### PR TITLE
TYP: Type annotations in pandas/io/sas/sas_constants.py

### PR DESCRIPTION
**Pull request by ShaharNaveh**
- [ ] closes #xxxx
- [ ] tests added / passed
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/io/sas/sas_constants.py [hunk 1]
```diff
@@ -1,106 +1,108 @@
...
-column_format_length_length = 2
-column_label_text_subheader_index_offset = 28
-column_label_text_subheader_index_length = 2
-column_label_offset_offset = 30
-column_label_offset_length = 2
-column_label_length_offset = 32
-column_label_length_length = 2
-rle_compression = b"SASYZCRL"
-rdc_compression = b"SASYZCR2"
+align_1_checker_value: bytes = b"3"
```
Comment (WillAyd) [inline pandas/io/sas/sas_constants.py]: I don't think anything from line 10 to 99 here needs to be changed; mypy will infer these from the right hand side of the assignment as is

File: pandas/io/sas/sas_constants.py [hunk 2]
```diff
@@ -112,18 +114,18 @@
...
-    row_size_index = 0
-    column_size_index = 1
-    subheader_counts_index = 2
-    column_text_index = 3
-    column_name_index = 4
-    column_attributes_index = 5
-    format_and_label_index = 6
-    column_list_index = 7
-    data_subheader_index = 8
+    row_size_index: int = 0
```
Comment (WillAyd) [inline pandas/io/sas/sas_constants.py]: Same comment on this block - I *think* should be inferred as is

**Review by WillAyd — CHANGES_REQUESTED** [review]

**Comment by MomIsBestFriend** [comment]
Oh, so only keep the annotations of ```Dict, List, Tuple```?

**Comment by WillAyd** [comment]
That's right

**Review by WillAyd — CHANGES_REQUESTED** [review]
@MomIsBestFriend actually I don't think any of the annotations are required as mypy would infer these anyway. 

You really only need to annotate containers when you create an empty one first like `data = []` or when  what is created doesn't match all of the possible types, i.e. `data: List[str, int] = [1, 2]` would require an annotation because mypy would infer data to be `List[int]` without annotation

**Comment by MomIsBestFriend** [comment]
Oh I see.

Closing this then, (sorry for the waste of time)

**Pull request closed without merging**